USAGE.md

```diff
@@ -35,17 +35,395 @@ Descent 3 Message(Error: Couldn't find the string table.)
+- The read-only base directories can only contain read-only files. There can be any number of read-only base directories. By default, Descent 3 uses zero read-only base directories.
+
+You can set the writable base directory and the list of read-only base directories using the `-setdir`, `-useexedir` and `-additionaldir` command-line options (see [the next section](#command-line-options)).
+
+When Descent 3 tries to find a read-only file, then it will look through the list of base directories in this order:
+
+- the last read-only base directory that was specified on the command-line,
+- the second-to-last read-only base directory that was specified on the command-line,
+- the third-to-last read-only base directory that was specified on the command-line,
+- …
+- the first read-only base directory that was specified on the command-line and, finally,
+- the writable base directory.
+
+Files that are in base directories that are higher on that list will override files that are in base directories that are lower on that list. For example, lets say that you run Descent 3 like this:
+
+```
+Descent3 -setdir /home/user/my-writable-base-directory -additionaldir /home/user/my-read-only-base-directory
+```
+
+Let’s also say that both `my-writable-base-directory` and `my-read-only-base-directory` contain a file named `d3.hog`. In this example, Descent 3 will load `/home/user/my-read-only-base-directory/d3.hog` because read-only directories have a higher precedence than the writable base directory. Descent 3 will ignore `/home/user/my-writable-base-directory/d3.hog`.
+
+## Command-Line Options
+
+The following command-line options are available in Descent 3. You can set command-line options on the Misc. tab of the Setup section of the Descent 3 launcher or by creating a shortcut to `Descent3.exe`. Case is not significant in command-line options, and `-`, `--`, and `+` are all accepted.
+
+<table>
+  <tr>
+    <th>Option</th>
+    <th>Type</th>
+    <th>Default</th>
+    <th>Platform</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <th colspan="5">Display Options</th>
+  </tr>
+  <tr>
+    <td><code>-aspect &lt;value&gt;</code></td>
+    <td>floating-point number</td>
+    <td>1.3333333333333333</td>
+    <td>all</td>
+    <td>Specifies the screen aspect ratio for non-standard displays, such as wide-screen TVs.</td>
+  </tr>
+  <tr>
+    <td><code>-fullscreen</code> or <code>-f</code></td>
+    <td>boolean</td>
+    <td>On</td>
+    <td>all</td>
+    <td>Run game in fullscreen mode.</td>
+  </tr>
+  <tr>
+    <td><code>-height &lt;height&gt;</code></td>
+    <td>integer</td>
+    <td>480 unless you select a different value in the options menu</td>
+    <td>all</td>
+    <td>Sets the screen resolution to the specified height, if possible.</td>
+  </tr>
+  <tr>
+    <td><code>-himem</code></td>
```

Comment:
Outside of the scope of this PR, but some of these old options could be cleaned up, they really are not relevant to the modern-age player

Reply:
I agree that it would be good to clean up some of the old command-line options, and I agree that it’s out of scope for this PR. To make sure that we don’t forget about that idea, I opened #638.